modin/experimental/pandas/test/test_io_exp.py

```diff
@@ -164,24 +168,25 @@ def test_read_single_csv_with_parse_dates(self, parse_dates):
     Engine.get() != "Ray", reason="Currently only support Ray engine for glob paths."
 )
 def test_read_multiple_csv_s3():
-    modin_df = pd.read_csv_glob("S3://noaa-ghcn-pds/csv/178*.csv")
-
-    # We have to specify the columns because the column names are not identical. Since we specified the column names, we also have to skip the original column names.
-    pandas_dfs = [
-        pandas.read_csv(
-            "s3://noaa-ghcn-pds/csv/178{}.csv".format(i),
-            names=modin_df.columns,
-            skiprows=[0],
-        )
-        for i in range(10)
-    ]
-    pandas_df = pd.concat(pandas_dfs)
-
-    # Indexes get messed up when concatting so we reset both.
-    pandas_df = pandas_df.reset_index(drop=True)
-    modin_df = modin_df.reset_index(drop=True)
+    path = "S3://modin-datasets/testing/multiple_csv/test_data*.csv"
 
-    df_equals(modin_df, pandas_df)
+    def _pandas_read_csv_glob(path, storage_options):
```

Comment:
Could we make this a helper function?

Reply:
Probably if would be better to create helper function like this `pandas_read_csv_glob` for using in all our `read_csv_glob` tests, because the most part of tests in `test_io_exp.py` emulate pandas `read_csv_glob` behavior in several places. It should be resolved as a separate issue because it requires a lot of code changes in `test_io_exp.py`

Reply:
Sounds reasonable to me. Could you create the issue for it?